test/test_suit/test_string_json.py

```diff
@@ -24,7 +24,7 @@ def send(self):
                 {'numbernumbernumbernumbernumbernumbernumbernumbernumbernumbernumbernumber': str(100)}
             ).encode('utf-8'))
         else:
-            value.append('')
```

Comment:
I understand we need use None for the tombstone tests, but should we have a test to test the empty string behavior instead of updating everything to None?

Reply:
added empty string test and comment to explain the differences between community vs custom converter record behavior